src/components/feed/index.js

```diff
@@ -6,32 +6,61 @@ import {LuShare} from "react-icons/lu"
 
 
 export function FeedItem(props){
-   
-    return(
-        <div>
-            <div className="flex">
+  
+  function formatDate(){
```

Comment:
you can move this function into a `utilities` directory and export from there

Reply:
Resolved